packages/nimble-components/src/dialog/specs/README.md

```diff
@@ -78,35 +80,74 @@ We will not make any special effort to provide forms support (i.e. form `type="d
 
 ### Anatomy
 
-Because we do not have concrete designs for Nimble dialogs, we will initially have a single, default slot and leave layout and styling of contents completely up to the client. One unfortunate consequence of this is that the component will not be able to enforce design coherence, like position, style, label casing, etc. of dialog buttons. This can be mitigated by implementation of a higher-level service that creates consistently-styled, common dialogs. When we have designs for the dialog, we can revisit this and try to apply some common styling and layout.
+The visual design spec has a few different layouts for the dialog. Not all layouts will be supported in the initial styling pass of the dialog. See below for details on what parts will and will not be supported in the intial styling pass:
+
+-   Title
+    -   Included in initial styling pass: Yes
+    -   Rationale: The title is a fundamental part of the dialog. Adding the title will also resolve an accessibility issue with the dialog's aria-label.
+    -   Additional details: The content provided for the title will be used as the label of the dialog for accessibility purposes.
+-   Subtitle
+    -   Included in initial styling pass: Yes
+    -   Rationale: The styling of the subtitle is straight-forward and there are no open questions associated with it.
+-   Warning message
+    -   Included in initial styling pass: No
+    -   Rationale: There isn't a pressing need for this part of the dialog yet, and there are a number of questions that need to be resolved prior to adding the messages:
+        -   Should the messages be arbitrary content provided by the user, or should we have an API around providing message(s) with a specified severity, icon, color, etc?
+        -   What, if any, is the overlap between these messages and a future `<nimble-banner>` component?
+        -   Do we allow multiple messages at the same time? If so, is there a maximum height the messages can consume within the dialog?
+-   Close button
+    -   Included in initial styling pass: No
+    -   Rationale: There isn't a pressing need for this component yet, and there are a number of questions that need to be resolved prior to adding the button:
+        -   Should the API allow a client to slot any arbitrary buttons, including a close button, or should we only support one single button that closes the dialog?
+        -   What control do we need to give over the close button? Do we need a way for the user to explicitly disable it?
+        -   Should there be any connection between the `prevent-dismiss` attribute on the dialog and the visibility or disabled state of the close button?
+-   Content
+    -   Included in initial styling pass: Yes
```

Comment:
do we figure out / have open questions about a dialog max height and if overflow content should scroll or the dialog grows to fit content?

Reply:
I talked with Brandon, and he has a max-height for the dialog now. The dialog grows to fit the content until the max-height is reached, and then it will scroll. There are some open questions about the styling details when the content is scrollable, but I think those can get resolved in the implementation.

I'll update the spec with this new information.